js/views/modals/orderDetail/summaryTab/Fulfilled.js

```diff
@@ -72,11 +72,31 @@ export default class extends BaseVw {
     return this;
   }
 
+  revealEscapeChars(input) {
+    const output = input.replace(/[<>&\n"]/g, x => ({
+      '<': '&amp;lt;',
+      '>': '&amp;gt;',
+      '&': '&amp;&',
+      '"': '&amp;quot;',
+      '\n': '<br />',
+    }[x]
+    ));
+
+    return output;
+  }
+
   render() {
+    const cryptocurrencyDelivery = this.dataObject.cryptocurrencyDelivery;
+    let transactionID = cryptocurrencyDelivery && cryptocurrencyDelivery[0] || {};
```

Comment:
Would a one-line accomplish the same thing here?

```
const transactionID = cryptocurrencyDelivery && cryptocurrencyDelivery[0] &&
  cryptocurrencyDelivery[0].transactionID || '';
```

I think the encoding could also be done when you pass the variable to the template, although I'd probably rename it to a clearer name, since the encoded version is distinct from what I'd consider the actual transaction id.

```
         ...this.dataObject || {},
         encodedTxId: this.revealEscapeChars(transactionID),
```

Reply:
- Made these changes to accompany the team's/Rob's style suggestions.